kia-cli/build.gradle.kts

```diff
@@ -39,7 +39,10 @@ tasks {
 
     named<ShadowJar>("shadowJar") {
         manifest {
-            attributes(mapOf("Main-Class" to "com.faforever.ice.KiaApplication"))
+            attributes(mapOf(
+                "Main-Class" to "com.faforever.ice.KiaApplication",
+                "Implementation-Version" to archiveVersion,
```

Comment:
archiveVersion? This variable is not declared anywhere?

Reply:
This is what faf-client [uses](https://github.com/FAForever/downlords-faf-client/blob/e3608b26c3d8d288da0f7019d77a4832ff62cd49/build.gradle#L193).  It's a gradle bundling/task [property](https://docs.gradle.org/current/kotlin-dsl/gradle/org.gradle.api.tasks.bundling/-abstract-archive-task/archive-version.html) that seems to get it's value from `version`. But probably clearer to just use `version` directly so I'll switch it to that.

Reply:
Could be a "magic" variable from the gradle groovy language. Also the faf client uses it in the jar task, we have it in the shadow jar task 🤔